### PR TITLE
use existing request requirements + fix some issues

### PR DESCRIPTION
**Pull request by davidsmejia**
## Issue Number

#355 

* Adds logic and components to use an existing resource's request requirements so you dont have to re-enter that
* Also adds the `Resource Added!` page to prevent an error from flashing before navigating to resource page
* fixes an issue with Popover's where clicking the link again wouldnt dismiss the popover
* also snuck in moving description in cell line to basic information

## Types of changes

- Bugfix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)

## Functional tests

Listed resources and selected existing resource to use the same requirements

## Checklist

- [x] Lint and unit tests pass locally with my changes

## Screenshots

Screen Shot 2020-10-15 at 4 31 16 PM

Screen Shot 2020-10-15 at 4 30 58 PM
Screen Shot 2020-10-15 at 4 40 59 PM

### CODE REVIEW

File: client/src/components/resources/RequestRequirementsForm.js [hunk 1]
```diff
@@ -53,142 +89,182 @@ export default () => {
...
                     />
+                    {getAttribute('sharer_pays_shipping') === false && (
+                      <Box pad={{ horizontal: 'large' }}>
+                        <Text
+                          italic
+                          size="small"
+                          color="black-tint-60"
+                          margin={{ bottom: 'small' }}
+                        >
+                          What payments methods you support? Choose at least 1.
```
Comment (Philinphiladelphia) [inline client/src/components/resources/RequestRequirementsForm.js:205]:
```suggestion
                          What payment methods do you support? Choose at least 1.
```

**Review by Philinphiladelphia — APPROVED** [review]
Looking good!

**Review by Philinphiladelphia — COMMENTED** [review]
This is not within the scope of this PR, but at some point we should consider creating user-friendly versions of the resource names so that it shows "Plasmid" instead of "PLASMID".

**Comment by davidsmejia** [comment]
I think that is accomplished with readableNames where did you see it appear as PLASMID?

**Comment by Philinphiladelphia** [comment]
> I think that is accomplished with readableNames where did you see it appear as PLASMID?

I just saw OTHER in your second screenshot. If that is indeed taken care of, then no worries.

**Comment by davidsmejia** [comment]
ah gotcha yeah that was just in the title i typed that when making the example resource/material

**Pull request merged**